client/components/ResponsiveFlex/ResponsiveFlex.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { PropsWithChildren, useEffect } from 'react';
+
+interface Props {
+  breakpoint: number;
+}
+
+function ResponsiveFlex({ breakpoint, children }: PropsWithChildren<Props>) {
+  const containerRef = React.createRef<HTMLDivElement>();
+
+  useEffect(() => {
+    if (breakpoint) {
+      if (document.head.querySelector(`[data-flex-breakpoint="${breakpoint}"]`)) {
```

Comment:
Let's try and make this a little more readable by moving this code into a separate function `classExists(breakpoint): boolean` that we'll call.